yt/yt/server/query_tracker/public.h

```diff
@@ -0,0 +1,37 @@
+#include <yt/yt/client/query_tracker_client/public.h>
```

Comment:
Why did you move these to public.h? They are more private than public as they are not used outside this module.

Reply:
Well, it differs from my understanding. I believe that private.h and public.h are both for lightweight declarations such as forward declarations or logger declarations. Then, you put any declaration in private.h if it is possible (i.e. it is not used in external modules), and public.h otherwise. Finally, private.h includes public.h.

In your interpretation it is hard to distinguish private.h from public.h at all.